src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -26,7 +26,7 @@
     public static final String MESSAGE_SUCCESS = "New entry added: %1$s";
     public static final String MESSAGE_UNDO_SUCCESS = "Undo add new entry: %1$s";
     public static final String MESSAGE_DUPLICATE_ENTRY = "This entry already exists in the todo list";
-    public static final String START_FLAG = "start/";
+    public static final String START_FLAG = "st/";
```

Comment:
Can you make an issue? this changes is from pulling. We shouldn't fix it here I think?